example/.dart_tool/package_config_subset

```diff
@@ -0,0 +1,285 @@
+_fe_analyzer_shared
```

Comment:
`example/.dart_tool` folder should be gitignored

Reply:
Done.